package.json

```diff
@@ -65,5 +72,9 @@
     "react-test-renderer": "^16.13.1",
     "tern": "^0.24.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "node-fetch": "^2.6.1",
+    "plist": ">=3.0.2",
+    "ua-parser-js": "^0.7.24"
```

Comment:
Not sure why this is needed anymore. Why not just delete it both as a a `dependency` and as a `resolution`?

Reply:
If you inspect the `yarn.lock` file, you can see that my statement is correct and that they can be deleted

Reply:
You can inspect `yarn.lock` to see if they are still needed

Reply:
not sure i understand. isn't the `yarn.lock` file generated based on the contents of `package.json` looks like the ua-parser is used by `fbjs`?